cpu/stm32_common/periph/uart.c

```diff
@@ -292,6 +327,19 @@ void uart_poweroff(uart_t uart)
 #endif
 }
 
+uint8_t _data_mask(uart_t uart)
```

Comment:
needs to be declared `static`

Reply:
further: I think it would be better to calculate these masks during configuration and save them in the context struct -> doing this evaluation in the **hot interrupt code path** seems not very nice to me.

Reply:
That can be done as well, I mean it isn't doing that much then you don't need to increase the size of the struct.